checks/encoding.py

```diff
@@ -115,5 +115,31 @@ def checkEncoding(lines):
             severity = LEVEL_WARNING
         else:
             severity = LEVEL_INFO
-        return [severity, "{}% Encoder Overload".format(val),
-                """Encoder overload may be related to your CPU or GPU being overloaded, depending on the encoder in question. If you are using a software encoder (x264) please see the <a href="https://obsproject.com/wiki/General-Performance-and-Encoding-Issues">CPU Overload Guide</a>. If you are using a hardware encoder (AMF, QSV/Quicksync, NVENC) please see the <a href="https://obsproject.com/wiki/GPU-overload-issues">GPU Overload Guide</a>."""]
+
```

Comment:
As a utility, I would have `hasx264` and `hasNvenc` variables at the start, so you can reuse the resuls from past search() functions. It'll make it slightly more readable too.